src/tiles/TileStyles.js

```diff
@@ -37,7 +37,7 @@ export const TileName = styled("div")(() => ({
   whiteSpace: "normal" /* Allow multiline text */,
   overflow: "hidden" /* Hide overflowing content */,
   textOverflow: "ellipsis" /* Add ellipsis for very long content */,
-  overflowWrap: "anywhere",
+  // overflowWrap: "anywhere",
```

Comment:
`overflowWrap: "anywhere"` css property here was preventing the text to overflow from the tile

<img width="192" alt="Screenshot 2023-06-13 at 11 13 16 PM" src="https://github.com/tejas-k3/tejas-kothari-portfolio/assets/91131867/629c8639-91d8-4039-a8d2-53a24122a75a">

While I know that this isn't the best solution, it was keeping the text confined within the tile itself till we deal with this issue through proper styling.